### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by gitd01**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Apart from that, you can check the Algorithms section and add an algorithm of yo
 - To add implementation of a Data Structure/ Algorithm navigate to the respective folder
 - If the Data Structure or Algorithm you want to add doesn't have a folder, you are free to create a new folder
 - Inside the respective folder, click `Create new file`
-- File name convention: DataStructure_languageName.extension or AlgorithmName_languageName.extension
+- File name convention: DataStructure.extension or AlgorithmName.extension
```
Comment (harshit-sharma-gits) [inline README.md:22]: Will you please add the following line after this:

`(See issue [#12](https://github.com/DecodersCommunity/dsa-library/issues/12) and [#13](https://github.com/DecodersCommunity/dsa-library/issues/12) for detailed instructions)`

**Review by harshit-sharma-gits — CHANGES_REQUESTED** [review]
I've added a comment, please check. And thanks for mentioning this documentation discrepancy.

File: README.md (hunk 1 above)
Comment (gitd01) [reply README.md:22]: sure

**Comment by harshit-sharma-gits** [comment]
I think this can be closed now

**Pull request closed without merging**